docs/modules/ROOT/pages/reference/extensions/core.adoc

```diff
@@ -423,3 +423,21 @@ The action to take when `CamelMain` encounters an unknown argument. fail - Print
 [.configuration-legend]
 {doc-link-icon-lock}[title=Fixed at build time] Configuration property fixed at build time. All other configuration properties are overridable at runtime.
 
+[NOTE]
+[id=duration-note-anchor-core]
+.About the Duration format
+====
+To write duration values, use the standard `java.time.Duration` format.
+See the link:https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/time/Duration.html#parse(java.lang.CharSequence)[Duration#parse() Java API documentation] for more information.
+
+You can also use a simplified format, starting with a number:
+
+* If the value is only a number, it represents time in seconds.\n
+* If the value is a number followed by `ms`, it represents time in milliseconds.\n
```

Comment:
Do these `\n` get rendered? Or is AsciiDoc / Antora smart enough to figure things out?

The GitHub rendering does include them in their raw form:

https://github.com/apache/camel-quarkus/blob/5bc3a136e1265326b7814afb975be327b3a5b214/docs/modules/ROOT/pages/reference/extensions/core.adoc

Reply:
Probably a leftover as other lines don't have this ?

Reply:
Good catch, we definitely do not need them here. Let me remove them.

I asked Guillaume, whether they are intentional at least in Quarkus templates.

Reply:
It's a mistake in Quarkus too. Thanks again, @jamesnetherton !